store_with_fallbacks_test.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package credentials
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"oras.land/oras-go/v2/registry/remote/auth"
+)
+
+func TestStoreWithFallbacks(t *testing.T) {
+	ns := &NativeStore{
+		programFunc: testCommandFn,
+	}
+	fs, err := NewFileStore("testdata/valid_auths_config.json")
+	if err != nil {
+		t.Fatal("NewFileStore() error =", err)
+	}
+	sf := NewStoreWithFallbacks(ns, fs)
```

Comment:
I think using a test store here is enough.

Reply:
Thanks, I totally forgot about test store lol

Reply:
Replaced with test stores.